satpy/etc/writers/cf.yaml

```diff
@@ -2,6 +2,6 @@ writer:
   name: cf
   description: Generic netCDF4/CF Writer
   writer: !!python/name:satpy.writers.cf_writer.CFWriter
-  filename: '{name}_{start_time:%Y%m%d_%H%M%S}.nc'
+  filename: '{platform_name}-{sensor}-{start_time:%Y%m%d%H%M%S}-{end_time:%Y%m%d%H%M%S}.nc'
```

Comment:
Not sure it was mentioned in the comments, but does anyone know/remember where this `-` separate filename came from? As you mentioned it seems to have existed in the reader first, but it isn't clear to me why/where from.

Reply:
I think my question here still stands, but not sure we're going to get anyone who has a real answer.